src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryUtil.java

```diff
@@ -73,6 +77,7 @@ public final class BigQueryUtil {
   private static final Logger LOG = LoggerFactory.getLogger(BigQueryUtil.class);
 
   private static final String DEFAULT_PARTITION_COLUMN_NAME = "_PARTITIONTIME";
+  private static final String BIGQUERY_BUCKET_PREFIX_PROPERTY_NAME = "io.cdap.plugin.bigquery.bucket.prefix";
```

Comment:
we do a similar thing for cmek, where the argument key is 'gcp.cmek.key.name'. Let's follow a similar pattern and name it something like 'gcp.bigquery.bucket.prefix'

Reply:
Updated